kolibri/core/assets/src/state/actions.js

```diff
@@ -156,16 +156,19 @@ function refreshBrowser(url) {
   window.location.href = url || window.location.origin;
 }
 
-function kolibriLogin(store, sessionPayload) {
+function kolibriLogin(store, sessionPayload, firstDeviceSignIn) {
```

Comment:
Maybe name this `isFirstDeviceSignIn` to match conventions for booleans in the code. A JSDoc comment above this function might be nice too.